src/components/Loading.js

```diff
@@ -13,7 +13,7 @@ const Loading = ({
         return (
             <Paper className="paper" style={{"height": "100vh"}}>
                 <Typography variant="h3">{loadingText}</Typography>
-                <img alt="loading penguin" src={newLoading} style={{fontSize:30, paddingTop:"25%"}}/>
+                <img alt="loading penguin" src={newLoading} style={{width:30, paddingTop:"25%"}}/>
```

Comment:
Alright, so im guessing you did some other styling because now its very small. Play around with it, look at the old pr and see whats missing.